lib/kiev/shoryuken/request_store.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Kiev
+  module Shoryuken
+    class RequestStore
```

Comment:
I think all of these classes should be wrapped into a `Middleware` module.

Reply:
Why? All of the classes there are middlewares. No point in introducing another level for a single entry.

Reply:
Btw. Not all of them are middlewares since you added `ContextReader` to the same folder.

Reply:
True enough. Moved them out.